### PR TITLE
Added pickupable object filter

### PR DESCRIPTION
**Pull request by deanwetherby**
Retrieval goal now only selects from pickupable objects for the target.

### CODE REVIEW

File: scene_generator/goals.py [hunk 1]
```diff
@@ -188,7 +188,7 @@ def __init__(self):
         super(EmptyGoal, self).__init__()
 
     def compute_objects(self, object_defs):
-        return [], []
+        return [], [], []
```
Comment (brianpippin) [inline scene_generator/goals.py:191]: I know Chris has this change in:  https://github.com/NextCenturyCorporation/MCS/pull/43/files

File: scene_generator/goals.py [hunk 2]
```diff
@@ -209,7 +209,8 @@ def __init__(self):
 
     def compute_objects(self, object_defs):
         # add objects we need for the goal
-        target_def = copy.deepcopy(random.choice(object_defs))
+        pickupables = [od for od in object_defs if 'pickupable' in od['attributes']]
+        target_def = copy.deepcopy(random.choice(pickupables))
```
Comment (brianpippin) [inline scene_generator/goals.py:213]: He's also modifying this line as well.  The changes look fine, but you will need to look for merge conflicts and check with him on how he's doing pickupable in MCS-145.

Comment (deanwetherby) [reply scene_generator/goals.py:213]: I didn't see #MCS-145. This PR is probably unnecessary then.

Comment (brianpippin) [reply scene_generator/goals.py:213]: It might be?  We just need @chrislong to chime in.

Comment (ThomasSchellenbergNextCentury) [reply scene_generator/goals.py:213]: Oh, whoops, sorry I missed this.  Yeah, I just posted a comment on his PR:  https://github.com/NextCenturyCorporation/MCS/pull/43/files#r408302841

We can still merge this into master so you two aren't blocked.  I'll merge it into his branch to resolve any conflicts.

**Pull request merged**